src/status_im2/contexts/quo_preview/links/url_preview.cljs

```diff
@@ -1,30 +1,18 @@
 (ns status-im2.contexts.quo-preview.links.url-preview
```

Comment:
FYI: The diff in this file is just to show in this PR how the abstraction is used. Nothing of importance changed.